blocks/count-cards/count-cards.js

```diff
@@ -0,0 +1,48 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
+  [...block.children].forEach((row) => {
+    const li = document.createElement('li');
+    while (row.firstElementChild) li.append(row.firstElementChild);
+    [...li.children].forEach((div) => {
+      if (div.children.length === 1 && div.querySelector('img')) {
+        div.classList.add(...['count-card-image']);
+        div.getElementsByTagName('img')[0].classList.add(...['icon-svg-ethos', 'lazyloaded']);
+      } else if (div.children.length === 3) {
+        div.querySelectorAll('h4')
+          .forEach((e) => {
+            e.classList.add(...['ethos-2-theme', 'heading-5', 'count-card-title']);
+          });
+        div.querySelectorAll('p')
+          .forEach((e) => {
+            e.classList.add(...['ethos-2-theme', 'body-2', 'count-card-text']);
```

Comment:
The `ethos-2-theme` class is already added to the `body` dom element. Do we really need to add it to the paragraphs too? Same for h4s above.

Reply:
fetched latest changes and removed this section of code.